nengo/spa/tests/test_compare.py

```diff
@@ -10,8 +10,8 @@ def test_basic():
     inputB = model.get_module_input('compare_B')
     output = model.get_module_output('compare')
     # all nodes should be acquired correctly
-    assert inputA[0] is model.compare.inputA
-    assert inputB[0] is model.compare.inputB
+    assert inputA[0] is model.compare.product.A
+    assert inputB[0] is model.compare.product.B
```

Comment:
Can't we use `model.compare.A` and `model.compare.B` here? That way, the test doesn't change if we change the internal implementation.